src/components/logsReact/logsReact.tsx

```diff
@@ -78,28 +97,66 @@ const JSONRPCLogger: React.FC<IProps> = (props) => {
   }
 
   return (
-    <div className={classes.logsReact}>
-      <div className={clsx(classes.extendDiv, drawerOpen && classes.hide)}>
-        <IconButton
-          aria-label="open filters"
-          onClick={handleDrawerOpen}
-          className={classes.menuButton}
-        >
-          <ChevronRightIcon />
-        </IconButton>
-      </div>
-      <MethodList
-        logs={props.logs}
-        active={methodFilter}
-        select={methodClick}
-        isDrawerOpen={drawerOpen}
-        closeDrawer={handleDrawerClose}
-      />
-      <div className={clsx(classes.content, {
-        [classes.contentShift]: drawerOpen,
-      })}>
-        <CardList logs={props.logs} filter={methodFilter} />
-      </div>
+    <div>
+      { sidebarAlignment === "left" ?
+        <div className={classes.logsReact}>
+          <div className={clsx(classes.extendDiv, drawerOpen && classes.hide)}>
+            <IconButton
```

Comment:
Why are these respective left/right child elements in different orders?

Reply:
To position the sidebar div correctly. For example, if the code for sidebarAlignment "right" was the same as "left" then the sidebar would slide in from the right but still appear on the leftside.

Reply:
Gotcha. Thanks. LGTM but what do I know ;)